src/components/chat/ModalContainer/ModalContainer.tsx

```diff
@@ -10,23 +10,29 @@ export default function ModalContainer({
   openModal,
   childrenElem,
   elemRef,
+  chatRoomUuid,
 }: {
   openModal: (content: ModalContentType) => void;
   childrenElem: ModalContentType;
   elemRef: any;
+  chatRoomUuid?: string | null;
```

Comment:
The type of the `chatRoomUuid` should be `string` because we only use this component in a chat list/chat and we know what the chat id should be. 

Change the type to a string in all places where you use chat id